src/BreathOfTheWild/Mods/MasterModeOptions/rules.txt

```diff
@@ -0,0 +1,111 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Master Mode Options"
+path = "The Legend of Zelda: Breath of the Wild/Mods/Master Mode Options"

```

Comment:
Feel like a better place for this graphic pack would be under Cheats since it affects gameplay directly. Although I guess we've got the day length graphic pack so hmm. If you do change it, do make sure to also move it to the appropriate folder.